src/apis/oracle.ts

```diff
@@ -18,12 +18,14 @@ export type OracleInfo = {
 
 export interface OracleAPI {
     getExchangeRate(): Promise<Big>;
+    getBtcTxFeesPerByte(): Promise<[number, number, number]>;
```

Comment:
You could return an object of type `BtcTxFeesPerByte` here

Reply:
`BtcTxFeesPerByte` inherits from `Struct`; is that the best way to handle it?
I just changed it to a quick custom interface exported from apis/oracle.ts, let me know what you think. Seems easier to me since it can be constructed on the fly, as it doesn't inherit all the Struct members. It does introduce yet another type definition, but if the return of the RPC changes the lib will need updating anyway, so I don't feel like that's a problem.

If you disagree I'm happy to change it, though!

Reply:
That's a much better approach indeed!